src/BcPrebidLoader.js

```diff
@@ -123,7 +162,11 @@ function apiInit() {
 
         if (!_isLoadedFromPage) {
             loadPrebidPlugin(path, runPlugin, handleError);
-        }
+		}
+		else {
+			registerPlugins(prebidPluginFunc, commandPluginFunc);
```

Comment:
This line is unnecessary - the plugin will already be registered if this line ever executes, making it redundant.

Reply:
Fixed.